src/main/java/io/phasetwo/service/resource/MembersResource.java

```diff
@@ -40,6 +44,29 @@ public Stream<UserRepresentation> getMembers(
         .searchForMembersStream(searchQuery, firstResult, maxResults)
         .map(m -> toRepresentation(session, realm, m));
   }
+  @GET
+  @Path("/with-roles")
```

Comment:
Won't take this one. Rationale is that it's only one additional call, and I don't want to pollute the API with convenience methods and objects.